brayns/Brayns.cpp

```diff
@@ -137,16 +107,15 @@ struct Brayns::Impl
         if(!geometryParameters.getCircuitConfiguration().empty() &&
             geometryParameters.getLoadCacheFile().empty())
             _loadCircuitConfiguration();
-
     }
 
     void render( const RenderInput& renderInput,
                  RenderOutput& renderOutput )
     {
         reshape( renderInput.windowSize );
 
-        _frameBuffer->map( );
-        _camera->set(
+        _engine->getFrameBuffer()->map( );
```

Comment:
map is OSPray centric. I think the initialization should be in the engine construction.